### ISSUE TITLE
Unable to compile on Overleaf.

### ISSUE THREAD

**Issue by TheLight8** [fmarotta/kaobook#289]
### Description
 
It just does not compile with any option on Overleaf.

### Minimal Working Example
 
Even just the example template in the source.

**Expected behavior:**
 
To compile, possibly with no error.

**Actual behavior:**
 
Does not compile, at all.

**Comment by cgosorio** [comment]
I suspect the problem is that lately Overleaf has reduced the compiling time for the free accounts. I have experience the very same problem with other templates that before was possible to use in Overleaf.

**Comment by fmarotta** [comment]
Indeed, unfortunately the problem seems to be compilation time. If you can, I would suggest using another editor such as TeXworks, TeXstudio, VScode, neovim, or emacs---they all have good support for LaTeX, and you could use GitHub as a "cloud" for collaborating with other people. I am planning to eventually rewrite kaobook from scratch making use of KOMA-script features that I didn't understand back then, so this should de-bloat the package and speed up the compilation.

Another option is to self-host an Overleaf instance; there are some tutorials online, but it's not for everybody. I will also look into hosting myself (on my raspberry pi or VPS) and see if I can offer the service for kaobook users (hopefully this doesn't violate any licences, but I am not sure).
